.travis.yml

```diff
@@ -1,11 +1,25 @@
+sudo: false
 language: java
 jdk:
   - oraclejdk8
+addons:
+  sauce_connect:
+    no_ssl_bump_domains: all
 env:
-  - NODE_VERSION=0.10
+  global:
+    - NODE_VERSION=0.12
+    - SAUCE_USERNAME="kaazing-build"
+    - SAUCE_ACCESS_KEY="f350bbbb-376e-4c0e-b7b6-ae78f58f67f2"
```

Comment:
How does this sauce lab key work.  Is it specific per each repo or generic for all kaazing.  Good either way, just want to understand